### PR TITLE
add the windmill-config-tox-tools job

### PR DESCRIPTION
**Pull request by goneri**
Depends-On: https://github.com/ansible-network/windmill-config/pull/622

### CODE REVIEW

File: .zuul.d/jobs.yaml [hunk 1]
```diff
@@ -46,3 +46,16 @@
...
+- job:
+    name: windmill-config-tox-tools
+    parent: tox
+    description: |
+      Runs the scripts from the tools directory
+
+      Uses tox with the ``tools`` environment.
+    vars:
+      tox_envlist: tools
+      test_setup_skip: true
```
Comment (pabelanger) [inline .zuul.d/jobs.yaml:59]: can be removed

File: .zuul.d/jobs.yaml [hunk 2]
```diff
@@ -46,3 +46,16 @@
...
+    files:
+      - nodepool/*.zuul.ansible.com.yaml
```
Comment (pabelanger) [inline .zuul.d/jobs.yaml:61]: should be regex, so

^nodepool

should be enough

Comment (pabelanger) [reply .zuul.d/jobs.yaml:61]: we also want to add tox.ini, as that is entry point and your tools script here.  So we test when both are also changed

File: .zuul.d/projects.yaml [hunk 3]
```diff
@@ -1,5 +1,11 @@
 ---
 - project:
+    check:
+      jobs:
+        - windmill-config-tox-tools
+    gate:
+      jobs:
+        - windmill-config-tox-tools
     templates:
       - ansible-python-jobs
```
Comment (pabelanger) [inline .zuul.d/projects.yaml:10]: lets keep templates first please

**Comment by goneri** [comment]
recheck

**Pull request merged**